packages/manager/src/features/PlacementGroups/PlacementGroupsDetail/PlacementGroupsLinodes/PlacementGroupsLinodesTableRow.tsx

```diff
@@ -46,6 +53,7 @@ export const PlacementGroupsLinodesTableRow = React.memo((props: Props) => {
       <TableCell actionCell>
         <InlineMenuAction
           actionText="Unassign"
+          disabled={isAllowedToEditPlacementGroup}
```

Comment:
The naming of this variable and the way that this util works feels confusing here.

`isAllowedToEditPlacementGroup` should return `true` if that linode has `read_write` permissions, so that makes sense. But then we'd be saying that the "Unassign" button should be disabled if the user has read_write abilities, and otherwise enabled? That sounds like it would be the opposite of what we want, so we're missing the `!`.

```suggestion
          disabled={!isAllowedToEditPlacementGroup}
```

Reply:
Correct, I will update.